tests/test_taskipy.py

```diff
@@ -247,6 +247,20 @@ def test_handling_sigint_according_to_subprocess_if_it_does_not_handle_it_gracef
 
         self.assertEqual(exit_code, 130)
 
+    def test_signals_should_be_sent_to_subprocess(self):
```

Comment:
suggestion, since we're only focusing on sigterm now:
```suggestion
    def test_sigterm_should_be_sent_to_subprocess(self):
```